sql/core/src/test/resources/sql-tests/results/join-empty-relation.sql.out

```diff
@@ -71,15 +71,15 @@ SELECT * FROM t1 LEFT SEMI JOIN empty_table
 -- !query schema
 struct<a:int>
 -- !query output
-
+1
```

Comment:
does it look right or not? LEFT SEMI join with empty table should has no output, right?

Reply:
Yes, it seems this change incorrect.